docs/desktop/gnome/valuta.md

```diff
@@ -0,0 +1,52 @@
+﻿---
+ title: Valuta
+ author: Christine Belzie
+ contributors:
+---
+
+## Introduction
+
+ If you often travel or are moving abroad, ease your finanical planning with Valuta. This application quickly converts currencies between two countries.
+
+## Assumptions
+
+This guide assumes you have the following:
+
+- Rocky Linux
+- Flatpak
+- FlatHub
+
+## Installation Process
+
+1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install** ![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)
+ 
```

Comment:
I think I'd move the screenshot attached to 1. (above) to it's own line above the numbered list item. Like this:
```
## Installation process

![Screenshot of the Valuta page on Flathub with the blue install button highlighted in a red square](images/01_valuta.png)

1. Go to [Flathub.org](https://flathub.org), type "Valuta" in the search bar and click  **Install**
```
That way you are consistent with the image above the option. (I think that's the way the  rest of them are anyway)